### PR TITLE
Bugfix/tab focus

### PR DESCRIPTION
**Pull request by srikary12**
This PR attempts to implement Tab Order as discussed in https://github.com/usebruno/bruno/issues/129

The Tab order stays inside Modal.

This is my first issue, please let me know if I miss something. 😄

### CODE REVIEW

**Comment by srikary12** [comment]
@helloanoop Can you review this and let me know if something needs to be added?

**Comment by helloanoop** [comment]
This is a QOL improvement. Will review and merge this post our 1.0.0 launch on Oct 28th.

File: packages/bruno-app/src/components/Modal/index.js [hunk 1]
```diff
@@ -78,10 +79,30 @@ const Modal = ({
   };
 
   useEffect(() => {
-    document.addEventListener('keydown', escFunction, false);
```
Comment (Theqwertypusher) [inline packages/bruno-app/src/components/Modal/index.js:81]: I'd recommend abstracting this to a custom hook that handles focus trap

File: packages/bruno-app/src/components/Modal/index.js [hunk 2]
```diff
@@ -78,10 +79,30 @@ const Modal = ({
...
+    const modalElement = modalRef.current;
+    const focusableElements = modalElement.querySelectorAll(
+      'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
+    );
+    const firstElement = focusableElements[0];
+    const lastElement = focusableElements[focusableElements.length - 1];
 
+    console.log(focusableElements);
```
Comment (Theqwertypusher) [inline packages/bruno-app/src/components/Modal/index.js:89]: remove log

**Comment by helloanoop** [comment]
Hey @srikary12 !

Thanks for working on this. We made some updates on top of your PR and the changes have been merged in #3075

Closing this PR

**Pull request closed without merging**